configs/ember.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+import ember from "eslint-plugin-ember";
+import eslintPluginEmberRecommended from "eslint-plugin-ember/configs/recommended";
+import javascript from "./javascript.js";
+
+export default [
+  ...javascript,
```

Comment:
do we want to include the `javascript` rules here by default, or allow developers to mix in the inheritance themselves?